doc/source/tokudb/tokudb_background_analyze_table.rst

```diff
@@ -0,0 +1,146 @@
+
+.. note:: 
+
+  If you're upgrading an existing |TokuDB| installation prior to :rn:`5.6.27-76.0`, the |TokuDB| plugin must be disabled and re-enabled via :program:`ps_tokudb_admin` to register this new plugin. Alternately the plugin can be enabled by running 
+  
+   .. code-block:: mysql 
+   
+     INSTALL PLUGIN tokudb_background_job_status SONAME 'ha_tokudb.so'
+
+Background Jobs
+===============
+
+Background jobs and schedule are transient in nature and are not persisted anywhere. Any currently running job will be terminated on shutdown and all scheduled jobs will be forgotten about on server restart. There can't be two jobs on the same table scheduled or running at any one point in time. If you manually invoke an ``ANALYZE TABLE`` that conflicts with either a pending or running job, the running job will be canceled and the users task will run immediately in the foreground. All the scheduled and running background jobs can be viewed by querying the :table:`TOKUDB_BACKGROUND_JOB_STATUS` table.
+
+New :variable:`tokudb_analyze_in_background` variable has been implemented in order to control if the ``ANALYZE TABLE`` will be dispatched to the background process or if it will be running in the foreground. 
+To control the function of ``ANALYZE TABLE`` a new :variable:`tokudb_analyze_mode` variable has been implemented. This variable offers options to cancel any running or scheduled job on the specified table (``TOKUDB_ANALYZE_CANCEL``), use existing analysis algorithm (``TOKUDB_ANALYZE_STANDARD``), or to recount the logical rows in table and update persistent count (``TOKUDB_ANALYZE_RECOUNT_ROWS``).
+
+``TOKUDB_ANALYSE_RECOUNT_ROWS`` is a new mechanism that is used to perform a logical recount of all rows in a table and persist that as the basis value for the table row estimate. This mode was added for tables that have been upgraded from an older version of |TokuDB| that only reported physical row counts and never had a proper logical row count. Newly created tables/partitions will begin counting logical rows correctly from their creation and should not need to be recounted unless some odd edge condition causes the logical count to become inaccurate over time. This analysis mode has no effect on the table cardinality counts. It will take the currently set session values for :variable:`tokudb_analyze_in_background`, and :variable:`tokudb_analyze_throttle`. Changing the global or session instances of these values after scheduling will have no effect on the job.
+
+Scheduled jobs that have not been started or are currently running can be canceled by setting the :variable:`tokudb_analyze_mode` to ``TOKUDB_ANALYZE_CANCEL`` and issuing the ``ANALYZE TABLE`` on the table for which you want to cancel all the jobs for.
```

Comment:
Hmm, the sounds kind of funny even though I probably originally wrote it.

Maybe it is better worded as:
```Any background job, both pending and running, can be canceled ...```